.docker/ci-testing/Dockerfile

```diff
@@ -3,8 +3,6 @@
 
 ARG ROS_DISTRO=foxy
```

Comment:
side note, should we change the default to rolling?

Reply:
Sure!

Reply:
Done!